sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Combine.java

```diff
@@ -2031,6 +2058,15 @@ public AccumT extractOutput(KV<K, Integer> key, AccumT accumulator, Context c) {
                   throws CannotProvideCoderException {
                 return accumCoder;
               }
+              @Override
+              public void populateDisplayData(DisplayData.Builder builder) {
+                super.populateDisplayData(builder);
+                builder.add(DisplayData.item("fanoutFn", hotKeyFanout.getClass())
+                    .withLabel("Fanout Function"));
+                if (hotKeyFanout instanceof HasDisplayData) {
```

Comment:
I would remove this. We take in `SerializableFunction` which doesn't itself implement `HasDisplayData`. I don't see any use-case for adding display data to the hotkey fun. If there is a use-case, then we should keep it.